.github/workflows/benchmarks.yml

```diff
@@ -44,6 +44,8 @@ jobs:
           # apply some patches for `pallets/gear/src/weights.rs`
           cp runtime/vara/src/weights/pallet_gear.rs pallets/gear/src/weights.rs
           sed -i -E 's/\w+::WeightInfo for SubstrateWeight/WeightInfo for SubstrateWeight/' pallets/gear/src/weights.rs
+          # generate code for lightweight scheduler that is used in gtest and other crates
+          ./scripts/weight-dump.sh
```

Comment:
I think this should be in `check.yml` because nobody will actually remember to manually update lightweight scheduler

Reply:
If you want to update weights for your branch this should be done via github actions workflow.